homeassistant/components/blue_current/strings.json

```diff
@@ -0,0 +1,117 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "api_token": "[%key:common::config_flow::data::api_token%]"
+        },
+        "description": "Enter your Blue Current api token",
+        "title": "Authentication"
+      }
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "limit_reached": "Request limit reached",
+      "invalid_token": "Invalid token",
+      "no_cards_found": "No charge cards found",
+      "already_connected": "IP is already connected",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+    }
+  },
+  "entity": {
```

Comment:
For all entity translations, please use `Title case` instead of `Whatever This Is Called`, don't even know if title case is the correct term, but you get the point.

Unless it's a brand name or something like that. Not the case here, but FYI